pkg/apiserver/web/router.go

```diff
@@ -30,5 +31,11 @@ func NewRouter() *echo.Echo {
 		AllowMethods: []string{echo.GET, echo.HEAD},
 	}))
 	e.Validator = NewValidator()
+	e.Use(middleware.RequestIDWithConfig(middleware.RequestIDConfig{
```

Comment:
Is the request_id printed automatically in every log or do we need to specify it?

Reply:
I saw on the print screen that it does. Nice!